modules/backend/constants/roles.ts

```diff
@@ -53,4 +53,4 @@ export const ROLES: RoleItem[] = [
   },
 ]
 
-export const getRoleByKey = (key: string) => ROLES.find((role) => role.key === key)
+export const getRoleByKey = (key?: string) => ROLES.find((role) => role.key === key)
```

Comment:
remove the bangd, in the usage of this `const role = getRoleByKey(roleKey!)!` => `const role = getRoleByKey(roleKey)` as discussed offline.